src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
   return (
     <tr>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
-      <td>{"...your code here"}</td>
+      <td>{props.transaction.posted_at}</td>
```

Comment:
if you like, you can avoid some repetition by pulling off the transaction into a variable. `const {transaction} = props`